src/main/java/Sam/commands/AddDeadlineCommand.java

```diff
@@ -0,0 +1,47 @@
+package sam.commands;
+
+import sam.constants.Message;
+import sam.services.Storage;
+import sam.services.TaskList;
+import sam.services.Ui;
+import sam.tasks.Deadline;
+import sam.tasks.Task;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+/**
+ * Adds a deadline to the task list.
+ */
+public class AddDeadlineCommand extends Command {
+
+    public static final String COMMAND_WORD = "deadline";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a deadline to the task list. \n"
```

Comment:
Nice constant names! Perhaps you could include some comments on what the constants are for?